CondFormats/CTPPSReadoutObjects/xml/mapping_timing_diamond_2017.xml

```diff
@@ -0,0 +1,129 @@
+<top>   
```

Comment:
Hi @franzoni 
As noted in https://github.com/cms-sw/cmssw/pull/18490#issuecomment-298711098, this PR only tackles the diamond detectors part (and not the pixel detectors using this `CTPPSPixelDAQMappingRcd` record). Hence, a follow-up is indeed expected for the migration of this xml mapping to the conditions db!